manifest.json

```diff
@@ -2,6 +2,7 @@
   "manifest_version": 2,
   "name": "Vimium",
   "version": "1.49",
+  "minimum_chrome_version": "41",
```

Comment:
@mrmr1993.  Thanks for this PR.  I'll take a proper look soon.

What aspect of the PR requires 41?  Is 40 not a bit too recent to be excluding?  It was just released in January (if I have it right).